File: FizzBang.py
count = 0

def FizzBang(i):
    if i%3 == 0 and i%5 == 0:
        print('FizzBang')
    elif i%3 == 0:
        print('Fizz')
    elif i%5 == 0:
        print('Bang')
    else:
        print(str(i))


while True:
    count += 1
    FizzBang(count)
    if count == 100:
        break

File: FizzBang.java
package fizzbang;

public class FizzBang {
    
    public static void main(String[] args) {
        for (int i = 1; i<=100; i++){
            if(i%3 == 0 & i%5 == 0){
                System.out.println("FizzBang");
            }else if(i%3 == 0){
                System.out.println("Fizz");
            } else if(i%5 == 0){
                System.out.println("Bang");
            }else{
                System.out.println(i);
            }
        }
    }
    
}
